src/goto-instrument/cover.cpp

```diff
@@ -193,14 +193,20 @@ Function: coverage_goalst::is_existing_goal
 
 \*******************************************************************/
 
-bool coverage_goalst::is_existing_goal(source_locationt source_location) const
+bool coverage_goalst::is_existing_goal(source_locationt source_loc) const
 {
   for(const auto &existing_loc : existing_goals)
   {
-    if(source_location.get_file()==existing_loc.get_file() &&
-       source_location.get_function()==existing_loc.get_function() &&
-       source_location.get_line()==existing_loc.get_line())
-      return true;
+    if(source_loc.get_file().compare(existing_loc.get_file())==0 &&
+      source_loc.get_function().compare(existing_loc.get_function())==0 &&
+      source_loc.get_line().compare(existing_loc.get_line())==0)
+    {
```

Comment:
For readability, multi-line conditions should be aligned (on the first character after the opening parenthesis).